start.sh

```diff
@@ -29,37 +9,19 @@ shutdown() {
        	exit 0
 }
 
-watch() {
```

Comment:
The htpasswd file will be updated automatically by Kubernetes, as it is mounted from a ConfigMap, so we don't need this watching.